manager/src/main/java/com/redhat/service/bridge/manager/models/Processor.java

```diff
@@ -29,7 +32,14 @@
         @NamedQuery(name = "PROCESSOR.findByBridgeIdAndName",
                 query = "from Processor p where p.name=:name and p.bridge.id=:bridgeId"),
         @NamedQuery(name = "PROCESSOR.findByStatusesAndShardId",
```

Comment:
I'd say now the signature is more something like `findByStatusesAndShardIdWithReadyDependencies`